File: 70-Climbing Stairs-爬楼梯/70-Climbing Stairs-爬楼梯.java
class Solution {
    public int climbStairs(int n) {
        int[] res = {1,1};
        for(int i = 2; i < n + 1; i++){
            int step = res[0] + res[1];
            res[0] = res[1];
            res[1] = step;
        }
        return res[1];
    }
}

File: 70-Climbing Stairs-爬楼梯/70-Climbing Stairs-爬楼梯.js
/**
 * @param {number} n
 * @return {number}
 */
var climbStairs = function(n) {
    let res = [1,1]
    for(let i = 2; i < n + 1; i++){
        const step = res[0] + res[1]
        res[0] = res[1]
        res[1] = step
    }
    return res[1]
};
